db/migrate/20161011151936_create_course_excluded_exercises.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCourseExcludedExercises < ActiveRecord::Migration
+  def change
+    create_table :course_excluded_exercises do |t|
+      t.integer       :sequence_number,         null: false
+      t.uuid          :excluded_uuid,           null: false
+      t.uuid          :course_uuid,             null: false
+
+      t.timestamps                              null: false
+    end
+  end
```

Comment:
There needs to be a uniqueness constraint on `:sequence_number` within the scope of `:course_uuid`:
```ruby
add_index :course_excluded_exercises, [:course_uuid, :sequence_number], unique: true
```